binaries/org.eclipse.swt.cocoa.macosx.aarch64/.settings/.api_filters

```diff
@@ -463,5 +1550,65 @@
                 <message_argument value="org.eclipse.swt.widgets.TypedListener"/>
             </message_arguments>
         </filter>
+        <filter id="338944126">
+            <message_arguments>
+                <message_argument value="org.eclipse.swt.widgets.TypedListener"/>
+                <message_argument value="getEventListener()"/>
+            </message_arguments>
+        </filter>
+        <filter id="338944126">
+            <message_arguments>
+                <message_argument value="org.eclipse.swt.widgets.TypedListener"/>
+                <message_argument value="handleEvent(Event)"/>
+            </message_arguments>
+        </filter>
+    </resource>
+    <resource path="Eclipse SWT/emulated/coolbar/org/eclipse/swt/widgets/CoolBar.java" type="org.eclipse.swt.widgets.CoolBar">
+        <filter id="336744520">
+            <message_arguments>
+                <message_argument value="@noextend"/>
+                <message_argument value="org.eclipse.swt.widgets.CoolBar"/>
+            </message_arguments>
+        </filter>
+    </resource>
+    <resource path="Eclipse SWT/emulated/coolbar/org/eclipse/swt/widgets/CoolItem.java" type="org.eclipse.swt.widgets.CoolItem">
+        <filter id="336744520">
+            <message_arguments>
+                <message_argument value="@noextend"/>
+                <message_argument value="org.eclipse.swt.widgets.CoolItem"/>
+            </message_arguments>
+        </filter>
+    </resource>
+    <resource path="Eclipse SWT/emulated/expand/org/eclipse/swt/widgets/ExpandBar.java" type="org.eclipse.swt.widgets.ExpandBar">
+        <filter id="336744520">
+            <message_arguments>
+                <message_argument value="@noextend"/>
+                <message_argument value="org.eclipse.swt.widgets.ExpandBar"/>
+            </message_arguments>
+        </filter>
+    </resource>
+    <resource path="Eclipse SWT/emulated/expand/org/eclipse/swt/widgets/ExpandItem.java" type="org.eclipse.swt.widgets.ExpandItem">
+        <filter id="336744520">
+            <message_arguments>
+                <message_argument value="@noextend"/>
+                <message_argument value="org.eclipse.swt.widgets.ExpandItem"/>
+            </message_arguments>
+        </filter>
+    </resource>
+    <resource path="Eclipse SWT/emulated/tooltip/org/eclipse/swt/widgets/ToolTip.java" type="org.eclipse.swt.widgets.ToolTip">
+        <filter id="336744520">
+            <message_arguments>
+                <message_argument value="@noextend"/>
```

Comment:
Looks like you are using older IDE build.
Please use https://download.eclipse.org/eclipse/downloads/drops4/I20240311-1800/ for your IDE

Reply:
Please use https://download.eclipse.org/eclipse/downloads/drops4/I20240311-1800/ for your IDE

And the baseline should be 4.31 RC2?

Reply:
The **baseline** should be 4.31 RC, but the **IDE** you are using should be https://download.eclipse.org/eclipse/downloads/drops4/I20240311-1800/ because of https://github.com/eclipse-pde/eclipse.pde/issues/1187

Reply:
BTW you can use fresh https://download.eclipse.org/eclipse/downloads/drops4/I20240313-0830/ if you like.

Reply:
I'm on it...

Reply:
New one force pushed. This time there were no API errors, only warnings.